docs/_documentations/installoncloud.md

```diff
@@ -12,95 +12,47 @@ parent: root
+  - For NFS, we recommend 777 permissions for the exported folders and ownership of `nobody:nogroup`
   - You do not need to set up the PV for local Kube, such as Minikube, Minishift, Docker Desktop, and others.
-- Set up the ClusterRole.
-  - **Important:** If you are installing Che with the `deploy_che.sh` script for OpenShift, do this step *after* installing Che.
+- Ensure cluster can pull images from `docker.io/eclipse`
+  - Both Eclipse Che and Eclipse Codewind hosts their docker images on `docker.io/eclipse`. Ensure that your cluster can pull from that registry and does not have PodSecurityPolicies blocking it from accessing dockerhub.
+- Set up the ClusterRole for Codewind.
   1. Clone the [Codewind Che plug-in repository](https://github.com/eclipse/codewind-che-plugin)
   2. Enter the `cd` command to go to the `codewind-che-plugin` repository.
   3. Run the `kubectl apply -f setup/install_che/codewind-clusterrole.yaml` command to create a cluster role with the required permission.
   4. Next, run the `kubectl apply -f setup/install_che/codewind-rolebinding.yaml` command.
 
-## Installing Che with the operator
+## Installing Che with chectl
 
-For instructions on installing Che with the Che operator, please see the [Eclipse Che operator on Operator Hub](https://operatorhub.io/operator/eclipse-che).
+We recommend using the `chectl` CLI to install Eclipse Che for use with Codewind. For instructions on installing the `chectl` CLI tool, please see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/)
 
-**Note:** Complete the following steps before you create your `CheCluster` custom resource.
+Once `chectl` has been installed:
 
-1. Create the `eclipse-codewind` cluster role: `kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/master/setup/install_che/codewind-clusterrole.yaml`.
+1. Download the [codewind-che checluster yaml](https://github.com/eclipse/codewind-che-plugin/blob/master/setup/install_che/che-operator/codewind-checluster.yaml)  file to your machine
+    - Feel free to make modifications to this file as you wish, but we recommend leaving the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and `spec.storage.preCreateSubPaths` field set to `true`
+2. Determine your ingress domain. If you're unsure of your ingress domain, ask your cluster administrator
+2. Install Che:
+    - On OpenShift 3.x run the `chectl server:start --platform=openshift --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=<codewind-che.yaml file>`
+    - On regular Kubernetes run: `chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=<codewind-che.yaml file>`
```

Comment:
- Can we just say "Kubernetes" instead of "regular Kubernetes"? I don't want to imply that the others are weird or something. 😅
- Also follow the format of line 48 if we can:
```
- On Kubernetes, run the following command:
<command here>
```